src/models/plot.ts

```diff
@@ -126,3 +129,21 @@ export const plotAreaFraction = 0.95;
 //NOTE: Consider moving this threshold to the useWindowSize hook if it will be utilized by other webapps.
 export const screenWidthThreshold = 1000;
 
+//When no domain is provided in url_pattern in traces then consider this as default domain to fetch data
+export const defaultDomain = '/dev.gudmap.org';
+
+//Alert messages
+export const invalidKeyAlert='Invalid key provided for hover template display pattern!';
```

Comment:
Similar to what we did in chaise, can you move these alert messages to a separate file called `message-map.ts`?

The file is in `src/utils/message-map.ts` in chaise.